src/io/flutter/logging/FlutterLogView.java

```diff
@@ -156,11 +156,12 @@ private DefaultActionGroup createPopupActionGroup() {
 
     @Override
     public boolean isSelected(AnActionEvent e) {
-      return logModel.shouldShowTimestamps();
+      return flutterLogPreferences.SHOW_TIMESTAMP;
```

Comment:
Again, this looks odd.  It reads like you're accessing a static final field...

I suppose I'd expect access (since it's in another class / file) to look like:

    flutterLogPreferences.shouldShowTimestamps();

Reply:
Oh, I use IDE generate get/set.
So it generated to `isShowTimestamps()`.
Should we change to `shouldShowTimestamps()` ?